<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>158</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nanonets</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="nanonetsDosyaYoluLine">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Tanıtılacak dosya yolu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nanonetsDosyaYoluBtn">
        <property name="text">
         <string>Dosya seç..</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="nanonetsTanitBtn">
      <property name="text">
       <string>Tanıt!</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="dbBtn">
      <property name="text">
       <string>Veritabanına gönder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nanonetsLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="nanonetsConfigAction"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="nanonetsConfigAction">
   <property name="text">
    <string>Nanonets Konfigürasyonu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
